### DISCUSSION TITLE
Custom fields for transitions

### DISCUSSION THREAD

**Discussion by harjoc** [ankitpokhrel/jira-cli#479] [Ideas]
Depending on the project settings, transitioning from "In Progress" to "Development Done" may require a custom Resolution field and/or a "Time spent" field, like the custom fields required when creating an issue.

**Comment by ankitpokhrel** [comment]
Related #472

**Comment by ankitpokhrel** [comment]
From [v1.2.0](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.2.0), you can now set `resolution`, `comment` and `assignee` when moving the issue
